web-app/src/app/store/saga/auth-saga.ts

```diff
@@ -80,10 +79,9 @@ function* logoutSaga({
   navigateTo: NavigateFunction;
 }>): Generator {
   try {
+    navigateTo(redirectScreen);
     yield app.auth().signOut();
     yield put(logoutSuccess());
-    yield put(anonymousLogin()); // Use anonymous login to keep the user signed in
```

Comment:
Anonymous login is not needed anymore here. Now anonymous login is being centralized in the root component.